### PR TITLE
Simplify and improve quickstart

### PR DESCRIPTION
**Pull request by steren**
* Do not use a callback for the quickstart example.

We assume the quickstart is not followed on GAE flex, but on a regular node runtime.
* Add the serviceContext 
* Add authentication steps, via api key

Add  a step to view the error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,39 +16,47 @@ applications running in almost any environment. Here's an introductory video:
...
-				console.log('Error reported!', res);
-			}
-		}
-	);
+	errorHandler.report(new Error('This is a test'));
 	```
 
+5. **View reported errors:**
+
+  Open Stackdriver Error Reporting (https://console.cloud.google.com/errors) to view the reported errors.
```
Comment (matthewloring) [inline README.md]: Prefer github link here

Comment (steren) [inline README.md]: I would like users of remember the URL of the Error Reporting UI, this is why I added a direct URL and not a markdown `[text](url)`.
Do you have something else in mind?

Comment (matthewloring) [inline README.md]: Ah ok. That makes sense.

File: README.md [hunk 2]
```diff
@@ -16,39 +16,47 @@ applications running in almost any environment. Here's an introductory video:
...
 
-	* [You can find the Error Reporting API here](https://cloud.google.com/error-reporting/)
+  [Enable the Error Reporting API here](https://console.cloud.google.com/apis/api/clouderrorreporting.googleapis.com/overview)
 
-2. **In your project, on the command line:**
+2. **Create an API key:**
+
+  (This authentication step is not needed if you run on Google Cloud Platform)
+
+  Follow [these instructions](https://support.google.com/cloud/answer/6158857?hl=en) to get an API key for your project.
```
Comment (ofrobots) [inline README.md]: Drop the `?hl=en` query string.

Comment (steren) [inline README.md]: done

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**